src/plugins/vuetify.js

```diff
@@ -69,6 +72,9 @@ export const dayjsLangs = {
     async es() {
         await import("dayjs/locale/es");
     },
+    async esES() {
```

Comment:
```suggestion
    async "es-ES"() {
```

Reply:
are you sure about that?

Reply:
lol, eslint doesn't allow that code

try:
```suggestion
    "es-ES": async () => {
```

Reply:
Okey works.